### PR TITLE
Add ever links package

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: localization/src/languages/ko.json [hunk 1]
```diff
@@ -200,6 +200,17 @@
...
+        "Send": {
+            "Amount": "Amount",
+            "ActionTitle": "Send tokens",
+            "Confirm": "Confirm",
+            "NetworkFee": "Network fee",
+            "NetworkFeeTilde": "~ ",
+            "RecipientAddress": "Recipient’s address",
+            "Title": "Confirm external action"
+        }
+    },
```
Comment (vetkamak) [inline localization/src/languages/ko.json]: @kkkhesha could you add korean translations here, pls?

File: localization/src/languages/ru.json [hunk 2]
```diff
@@ -174,6 +174,17 @@
...
+        "Send": {
+            "Amount": "Amount",
+            "ActionTitle": "Send tokens",
+            "Confirm": "Confirm",
+            "NetworkFee": "Network fee",
+            "NetworkFeeTilde": "~ ",
+            "RecipientAddress": "Recipient’s address",
+            "Title": "Confirm external action"
+        }
+    },
```
Comment (vetkamak) [inline localization/src/languages/ru.json]: @kkkhesha and also make translations here

**Review by sertony — APPROVED** [review]

File: localization/src/languages/ko.json [hunk 3]
```diff
@@ -200,6 +200,17 @@
     "Edit": "수정하기",
     "Error": "오류",
     "Events": "이벤트",
+    "EverLinks": {
+        "Send": {
+            "Amount": "Amount",
```
Comment (kkkhesha) [inline localization/src/languages/ko.json]:
```suggestion
            "Amount": "총액",
```

File: localization/src/languages/ko.json [hunk 4]
```diff
@@ -200,6 +200,17 @@
...
+            "ActionTitle": "Send tokens",
```
Comment (kkkhesha) [inline localization/src/languages/ko.json]:
```suggestion
            "ActionTitle": "토큰 보내기",
```

File: localization/src/languages/ko.json [hunk 5]
```diff
@@ -200,6 +200,17 @@
...
+            "Confirm": "Confirm",
```
Comment (kkkhesha) [inline localization/src/languages/ko.json]:
```suggestion
            "Confirm": "확인",
```

File: localization/src/languages/ko.json [hunk 6]
```diff
@@ -200,6 +200,17 @@
...
+            "NetworkFee": "Network fee",
```
Comment (kkkhesha) [inline localization/src/languages/ko.json]:
```suggestion
            "NetworkFee": "네트워크 수수료",
```

File: localization/src/languages/ko.json [hunk 7]
```diff
@@ -200,6 +200,17 @@
...
+            "NetworkFeeTilde": "~ ",
+            "RecipientAddress": "Recipient’s address",
```
Comment (kkkhesha) [inline localization/src/languages/ko.json]:
```suggestion
            "RecipientAddress": "수신자 주소",
```

File: localization/src/languages/ko.json [hunk 8]
```diff
@@ -200,6 +200,17 @@
...
+            "Title": "Confirm external action"
```
Comment (kkkhesha) [inline localization/src/languages/ko.json]:
```suggestion
            "Title": "외부 행동 확인"
```

File: localization/src/languages/ru.json [hunk 9]
```diff
@@ -174,6 +174,17 @@
     "Edit": "Изменить",
     "Error": "Ошибка",
     "Events": "События",
+    "EverLinks": {
+        "Send": {
+            "Amount": "Amount",
```
Comment (kkkhesha) [inline localization/src/languages/ru.json]:
```suggestion
            "Amount": "Сумма",
```

File: localization/src/languages/ru.json [hunk 10]
```diff
@@ -174,6 +174,17 @@
...
+            "ActionTitle": "Send tokens",
```
Comment (kkkhesha) [inline localization/src/languages/ru.json]:
```suggestion
            "ActionTitle": "Отправить токены",
```

File: Example/package.json [hunk 11]
```diff
@@ -56,6 +56,7 @@
         "@tonlabs/uistory.browser": "*",
         "@tonlabs/uistory.chats": "*",
         "@tonlabs/uistory.stickers": "*",
+        "@tonlabs/uistory.ever-links": "*",
```
Comment (SAnatoliiS) [inline Example/package.json:59]: It is necessary to add an example of using new components to the Example.

File: localization/src/languages/ru.json [hunk 12]
```diff
@@ -174,6 +174,17 @@
...
+            "Confirm": "Confirm",
```
Comment (kkkhesha) [inline localization/src/languages/ru.json]:
```suggestion
            "Confirm": "Подтвердить",
```

File: localization/src/languages/ru.json [hunk 13]
```diff
@@ -174,6 +174,17 @@
...
+            "NetworkFee": "Network fee",
```
Comment (kkkhesha) [inline localization/src/languages/ru.json]:
```suggestion
            "NetworkFee": "Комиссия сети",
```

File: localization/src/languages/ru.json [hunk 14]
```diff
@@ -174,6 +174,17 @@
...
+            "NetworkFeeTilde": "~ ",
+            "RecipientAddress": "Recipient’s address",
```
Comment (kkkhesha) [inline localization/src/languages/ru.json]:
```suggestion
            "RecipientAddress": "Адрес получателя",
```

File: stories/everLinks/src/SendSheet.tsx [hunk 15]
```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import { StyleSheet, View } from 'react-native';
+import type BigNumber from 'bignumber.js';
+
+import { uiLocalized } from '@tonlabs/localization';
+
+import { UIBoxButton } from '@tonlabs/uikit.controls';
+import { UICardSheet } from '@tonlabs/uikit.popups';
+import { UIConstant, UIStyle } from '@tonlabs/uikit.core';
```
Comment (SAnatoliiS) [inline stories/everLinks/src/SendSheet.tsx]: We are getting rid of legacy packages. For layout tasks, use UILayoutConstant from `@tonlabs/uikit.layout`

File: localization/src/languages/ru.json [hunk 16]
```diff
@@ -174,6 +174,17 @@
...
+            "Title": "Confirm external action"
```
Comment (kkkhesha) [inline localization/src/languages/ru.json]:
```suggestion
            "Title": "Подтвердить внешнюю операцию"
```

**Review by SAnatoliiS — CHANGES_REQUESTED** [review]

**Review by sertony — APPROVED** [review]

File: Example/src/screens/EverLinks.tsx [hunk 17]
```diff
@@ -0,0 +1,54 @@
...
+                        closeSendSheet();
+                    },
+                    signChar: 'EVER',
+                }}
+            />
+        </>
+    );
+}
+
+export function EverLinksScreen() {
```
Comment (SAnatoliiS) [inline Example/src/screens/EverLinks.tsx]: This is redundant. This was made by `Split.Screen` in `App.tsx`. Consider to remove `ExampleScreen` and `ExampleSection`.

**Review by SAnatoliiS — APPROVED** [review]

**Review by sertony — APPROVED** [review]

**Pull request merged**